CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 # OSTree is mandatory for aktualizr-lite
 option(BUILD_OSTREE "Set to ON to compile with ostree support" ON)
-option(BUILD_DOCKERAPP "Set to ON to compile with package manager support of docker-app" OFF)
+option(BUILD_DOCKERAPP "Set to ON to compile with package manager support of docker-app" ON)
+# always build with DOCKERAPP for aktualizr-lite, optional only for aktualizr
+set(BUILD_DOCKERAPP ON)
```

Comment:
Is there any need to keep this around?

Also - we have to remember that the update to our meta-updater will require removing this as well.

Reply:
I think we need BUILD_DOCKERAPP until the docker app manager implementation is in libaktualizr.